var/spack/repos/builtin/packages/r-testthat/package.py

```diff
@@ -14,17 +14,20 @@ class RTestthat(RPackage):
     url      = "https://cloud.r-project.org/src/contrib/testthat_1.0.2.tar.gz"
     list_url = "https://cloud.r-project.org/src/contrib/Archive/testthat"
 
+    version('2.3.2', sha256='1a268d8df07f7cd8d282d03bb96ac2d96a24a95c9aa52f4cca5138a09dd8e06c')
     version('2.2.1', sha256='67ee0512bb312695c81fd74338bb8ce9e2e58763681ddbcdfdf35f52dfdb0b78')
     version('2.1.0', sha256='cf5fa7108111b32b86e70819352f86b57ab4e835221bb1e83642d52a1fdbcdd4')
     version('1.0.2', sha256='0ef7df0ace1fddf821d329f9d9a5d42296085350ae0d94af62c45bd203c8415e')
 
     depends_on('r@3.1:', type=('build', 'run'))
     depends_on('r-digest', type=('build', 'run'))
+    depends_on('r-ellipsis', when='@2.0.0:', type=('build', 'run'))
+    depends_on('r-pkgload', type=('build', 'run'))
```

Comment:
This wasn't a dependency for 1.0.2, didn't check the other versions yet.

Reply:
`r-ellipsis` & `r-pkgload` got into dependency list from latest version which I added into package.